usage/installer/src/main/assembly/files/blueprints/seaclouds-on-byon.yaml

```diff
@@ -42,37 +51,24 @@ services:
       deployerPort: $brooklyn:component("deployer").attributeWhenReady("brooklynnode.webconsole.httpPort")
       deployerUser: $brooklyn:component("deployer").config("brooklynnode.managementUser")
       deployerPassword: $brooklyn:component("deployer").config("brooklynnode.managementPassword")
-      monitorHost: $brooklyn:component("monitoring").component("child","manager").attributeWhenReady("host.address")
-      monitorPort: $brooklyn:component("monitoring").component("child","manager").attributeWhenReady("modaclouds.mm.port")
-      slaHost: $brooklyn:component("sla-core").attributeWhenReady("host.address")
-      slaPort: $brooklyn:component("sla-core").attributeWhenReady("http.port")
-    launch.latch: $brooklyn:component("sla-core").attributeWhenReady("service.isUp")
-
-  - serviceType: brooklyn.entity.basic.SameServerEntity
-    name: SeaClouds SLA Service
-    brooklyn.children:
-    - serviceType: brooklyn.entity.webapp.tomcat.TomcatServer
-      name: SLA Core
-      id: sla-core
-      brooklyn.config:
-        java.sysprops:
-            DB_URL: >
-                $brooklyn:formatString("jdbc:%s%s",
-                component("sla-db").attributeWhenReady("datastore.url"), "sc_sla")
-            DB_USERNAME: "atossla"
-            DB_PASSWORD: "_atossla_"
-            MONITOR_METRICS_URL: >
-                $brooklyn:formatString("%s/v1/metrics", 
-                component("monitoring").component("child", "manager").attributeWhenReady("main.uri"))
-            SLA_URL: >
-                $brooklyn:formatString("http://%s:%s", 
-                component("sla-core").attributeWhenReady("host.address"), 
-                component("sla-core").attributeWhenReady("http.port"))
-      war: https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=eu.seaclouds-project&a=sla-service&v=LATEST&e=war
-      install.latch: $brooklyn:component("sla-db").attributeWhenReady("service.isUp")
+      slaHost: $SLA_HOST
```

Comment:
I'm wondering why is this necessary? Is it any conflict with `attributeWhenReady` between `SameServerEntity` children?